handbook/demand/README.md

```diff
@@ -114,7 +114,19 @@ Any changes to the current running ads visible to a user, including designs, key
 
 > **Do changes to keywords or targeting require a design review?** Currently, all changes to these things require discussion with our product marketer.
 
-
+### Posting on social media
+Social media posts should represent the Fleet brand in an appropriate tone, be on message, and encourage engagement from the community.
```

Comment:
```suggestion
Social media posts should represent the Fleet brand in an appropriate tone, be on message, and encourage engagement from the community. Each platform serves the community in a different way and our posts should fit that platform's tone. Social media posts should highlight:
```
These lines should be combined, no need for the first 2 bullets below as they don't represent steps to be followed.